src/desert/_make.py

```diff
@@ -130,7 +130,7 @@ def class_schema(
             attributes[field.name] = field_for_schema(
                 hints.get(field.name, field.type),
                 _get_field_default(field),
-                field.metadata,
+                {**meta, **field.metadata},
```

Comment:
Because we're using `field.metadata` second here, any metadata params in the child's nested field won't be overwritten.